src/pages/proposals/proposals.tsx

```diff
@@ -66,7 +66,7 @@ const Proposals = () => {
   ];
 
   // Only display this message during the genesis epoch
-  if (createNewProposal?.genesisEpochOver) {
+  if (!createNewProposal?.genesisEpochOver) {
```

Comment:
I remember reviewing this line, but didn't notice this :/ Shame on me :/

Reply:
Shame on for me implementing it 😅